bin/testEnvRootMinikube.sh

```diff
@@ -47,4 +47,4 @@ case "$1" in
     start) startMinikubeNone ;;
     stop) stopMinikube ;;
     wait) waitMinikube ;;
-esac
\ No newline at end of file
+esac
```

Comment:
don't you need to update the docker images that we have as well? They are all using k8s 1.7

Reply:
Ya most likely, will try that now